tools/export/nb/__init__.py

```diff
@@ -0,0 +1,402 @@
+import os
+import copy
+
+from os.path import relpath, join, exists, dirname, basename
+from os import makedirs
+from json import load
+
+from tools.export.exporters import Exporter, apply_supported_whitelist
+from tools.targets import TARGET_MAP
+from tools.utils import NotSupportedException
+from tools.build_api import prepare_toolchain
+
+POST_BINARY_WHITELIST = set([
+    "TEENSY3_1Code.binary_hook",
+    "MCU_NRF51Code.binary_hook",
+    "LPCTargetCode.lpc_patch",
+    "LPC4088Code.binary_hook"
+])
+
+
+class GNUARMNetbeans(Exporter):
+    NAME = 'GNU ARM Netbeans'
+    TOOLCHAIN = 'GCC_ARM'
+
+    @classmethod
+    def is_target_supported(cls, target_name):
+        target = TARGET_MAP[target_name]
+        return apply_supported_whitelist(
+            cls.TOOLCHAIN, POST_BINARY_WHITELIST, target)
+
+    @staticmethod
+    def prepare_sys_lib(libname):
+        return "-l" + libname
+
+    def toolchain_flags(self, toolchain):
```

Comment:
This is the same as the `Exporter.flags` attribute. Is there some reason that you could not use that?

Reply:
Well, I just done this, because in 'tools/exporter/gnuarmeclipse/__init.py' it is the same: https://github.com/cmens23/mbed-os/blob/e423947af35a024abdf12cdadce077810cf52398/tools/export/gnuarmeclipse/__init__.py#L78